### ISSUE TITLE
Require committer access on either end of issue transfers

### ISSUE THREAD

**Issue by daniel-beck** [timja/github-comment-ops#58]
### What feature do you want to see added?

Issue transfers should be limited to users who are committers on either source or destination repository.

**Comment by timja** [comment]
Do you think that org membership would be enough? or that only a maintainer of the component should be allowed to move the issue?

(the built-in github requirement is that you need write permissions on both sides which is a bit onerous in our community)

**Comment by daniel-beck** [comment]
> (the built-in github requirement is that you need write permissions on both sides which is a bit onerous in our community)

Right, hence my suggestion to relax it to one side of the transfer. The use case I can think of would be "pulling" an issue into my repo, or "pushing" it out of my repo. Perhaps allow the creator to move it too to cover badly created issues, that should cover most cases.
